src/main/java/net/sf/jabref/external/push/AbstractPushToApplication.java

```diff
@@ -32,23 +33,19 @@
 /**
  * Class for pushing entries into TexMaker.
  */
-public class PushToTexmaker implements PushToApplication {
-
-    private boolean couldNotCall;
-    private boolean notDefined;
-    private JPanel settings;
-    private final JTextField texmakerPath = new JTextField(30);
-    private final JTextField citeCommand = new JTextField(30);
+public abstract class AbstractPushToApplication implements PushToApplication {
 
+    protected boolean couldNotCall = false;
+    protected boolean notDefined = false;
+    protected JPanel settings;
+    protected final JTextField Path = new JTextField(30);
+    protected String searchPath = null;
+    protected String searchPathPreferenceKey = null;
+    protected String citeCommand = Globals.prefs.get(JabRefPreferences.CITE_COMMAND);
```

Comment:
previoulsy CITE_COMMAND_TEXMAKER was used. Are you sure this can be replaced by a generic CITE_COMMAND?